cmd/minikube/cmd/start.go

```diff
@@ -113,6 +115,8 @@ func runStart(cmd *cobra.Command, args []string) {
 	libMachineClient := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer libMachineClient.Close()
 
+	setProxy()
```

Comment:
@hferentschik As I described  https://github.com/minishift/minishift/pull/365#discussion_r99322076 I think we shouldn't add any our own param here because this is coming from docker-machine side.